From 91dbb2a09e007bb0010f618a4a075938425f5f39 Mon Sep 17 00:00:00 2001
From: Yi Kong <yikong@google.com>
Date: Fri, 19 Feb 2021 17:48:33 +0800
Subject: [PATCH 7/8] surfaceflinger: Enable PGO

This CL is no-op without the actual profile checked in.

Bug: 175205560
Bug: 180728514
Test: build
Change-Id: Ic62de6b423b0c7f96ad6475378bcc74440e93dbf
---
 services/surfaceflinger/Android.bp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/services/surfaceflinger/Android.bp b/services/surfaceflinger/Android.bp
index 4b56f8f..b65ba45 100644
--- a/services/surfaceflinger/Android.bp
+++ b/services/surfaceflinger/Android.bp
@@ -120,6 +120,10 @@ cc_defaults {
     lto: {
         thin: true,
     },
+    pgo: {
+        sampling: true,
+        profile_file: "surfaceflinger/surfaceflinger.profdata",
+    },
     // TODO(b/131771163): Fix broken fuzzer support with LTO.
     sanitize: {
         fuzzer: false,
-- 
2.31.1

